lib/mrsk/cli/healthcheck.rb

```diff
@@ -18,6 +18,7 @@ def perform
           if status == "200"
             info "#{target} succeeded with 200 OK!"
           else
+            error capture_with_info(*MRSK.healthcheck.logs)
```

Comment:
We already print the logs on the other rescue. Better to change the raise to be a specific, but unique exception, then catch that in the clause that prints the logs as well. Can add a dedicated HealthCheckError exception.

Reply:
Good point 👍. https://github.com/mrsked/mrsk/pull/54/commits/689b81014bb5914c9f45f14e18ed0c4731bee6a5